operators/src/adapters/raster_conversion.rs

```diff
@@ -37,9 +48,9 @@ where
         ctx: &'b dyn QueryContext,
     ) -> Result<BoxStream<'b, Result<Self::Output>>> {
         let stream = self.query_processor.query(query, ctx).await?;
+        let converted_stream =
+            stream.and_then(move |tile| process_tile(tile, ctx.thread_pool().clone()));
```

Comment:
We should use the method from https://github.com/geo-engine/geoengine/pull/457
I hope Michael merges is soon

Reply:
[21b9fa7](https://github.com/geo-engine/geoengine/pull/462/commits/21b9fa72a4b8ef5772a0993e1f11d2eaa23da869)